feedback/forms.py

```diff
@@ -181,3 +182,38 @@ class ContextTagForm(forms.ModelForm):
     class Meta:
         model = ContextTag
         fields = ['question_key', 'response_value', 'color', 'content']
+
+
+class ImportTagsForm(forms.Form):
+    source_course = forms.ModelChoiceField(queryset=None, label=_("Source course"))
```

Comment:
It does seem smart to have the course selection as a dropdown. However, after the source course has been selected, there is no preview at all of what tags the course has, which I think is a usability issues, especially considering that the current implementation allows staff to import tags from also courses they don't have access to (I think it may be a good idea to allow importing tags from indeed other teacher's courses, etc. but especially in those cases it would be extremely valuable to preview the tags before importing them and probably also select to import only a part of them). So I might reconsider how this process proceeds.
(However, this would probably be a bit more difficult to implement, and could be acceptable now to implement this in this simple fashion, and then later on change this when there's time.)